wsd-frontend/src/components/wsd/NewPost/client/NewPostForm.tsx

```diff
@@ -165,7 +186,7 @@ export default function NewPostForm({ categories }: { categories: APIType<'PostC
                     <div className="p-4 bg-muted rounded-full">
                       <Icons.Image className="w-8 h-8 text-muted-foreground" />
                     </div>
-                    <p className="font-medium">Choose a photo to upload</p>
+                    <p className="font-medium">Choose a photo to upload or paste from clipboard</p>
```

Comment:
I guess we need a new message for selecting + drag and drop + pasting here

Reply:
I adjusted the text, let me know if it is good?